pymc3/distributions/continuous.py

```diff
@@ -1776,8 +1776,15 @@ def logp(self, value):
             0 < self.kappa,
         )
 
+class LogNormal(LogNormal):
```

Comment:
@twiecki made the alias, but stupidly named the deprecated one LogNormal too, will correct it